### DISCUSSION TITLE
Browsing mode: No Anime, No furry

### DISCUSSION THREAD

**Discussion by radry** [civitai/civitai#299] [Ideas]
Please add to the Browsing mode the options for "no Anime" and "no Furry". Currently it's only possible to select "Safe", "Adult" and "Everything". The Category selection on top also only allows a single option and not multiple, thus it's currently impossible to hide only Anime.

**Comment by r7vz9h3** [comment]
You can hide all models with a certain tag in your profile settings, just add "furry" and "anime":
![image](https://user-images.githubusercontent.com/116812064/221419945-64554b37-8f7c-473f-a0dd-14281bd9e8f3.png)

**Reply by radry** [reply]
Ok that's a start but it's not very flexible to change it in ones profile. Maybe sometimes one want to temporarly hide them same like hiding adult content...

**Reply by WASasquatch** [reply]
> Ok that's a start but it's not very flexible to change it in ones profile. Maybe sometimes one want to temporarly hide them same like hiding adult content...

I do agree, adding "permanent" filters via account settings is unintuitive, and this is usually a dynamic on-the-go thing that could just be saved via user account settings or stored on the users browser.

**Reply by chromesun** [reply]
Even with the Hidden Tags option... there seems to be an anime tsunami of models that are completely untagged or miss out the anime tag.

**Reply by WASasquatch** [reply]
> Even with the Hidden Tags option... there seems to be an anime tsunami of models that are completely untagged or miss out the anime tag.

Maybe they should try the opencv anime cascade to detect anime faces.
